satat_backend/satat_backend/decoder/views.py

```diff
@@ -21,7 +21,26 @@ def ccsds_decoder(request):
     summary_df = summarize_data(data_df)
 
     # decoded_values = packetiser(data_df, summary_df)
-    decoded_values = get_packet_by_index(data_df, summary_df, 1)
-    if decoded_values["CCSDSAPID"] ==1:
-        new_hk_packet = HkPacket.objects.create(Filename = file.name, **decoded_values)
-    return JsonResponse(decoded_values, safe = False)
\ No newline at end of file
+    decoded_values_list = []  #Empty list to store decoded values
+
+    for i in range(1,len(summary_df)):
+        decoded_values = get_packet_by_index(data_df, summary_df, i)
```

Comment:
why we are accessing every value one by one her? we could have made dataframe directly or atleast an iterator to do this fast. @HarshadaRaut234  kindly look for some alternative